db/migrate/20190814021235_remove_organisation_users.rb

```diff
@@ -0,0 +1,23 @@
+class RemoveOrganisationUsers < ActiveRecord::Migration[5.2]
+  def up
+    drop_table(:organisation_users)
```

Comment:
🙋‍♀️ Is it too soon to drop this - should I just leave it in here silently for a while?

Reply:
do it!

Reply:
shall we truncate the shares table too?

Reply:
Yes, let's

Reply:
I have ended up migrating it properly after all - so I've added the `OrganisationUser` model back and written a migration for it. I'll do a second PR to remove these things after this version has been deployed